vizro-core/tests/unit/vizro/models/_components/test_table.py

```diff
@@ -170,27 +137,46 @@ def test_pre_build_actions_underlying_table_id(self, dash_data_table_with_id, fi
 
 
 class TestBuildTable:
-    def test_table_build_mandatory_only(self, standard_dash_table, expected_table):
-        table = vm.Table(
-            id="text_table",
-            figure=standard_dash_table,
-        )
-
+    def test_table_build_mandatory_only(self, standard_dash_table):
+        table = vm.Table(id="text_table", figure=standard_dash_table)
         table.pre_build()
+        table = table.build()
```

Comment:
I've actually asked her to rename this to` table` so we are consistent with the rest of the tests (dropdown, checklist, slider etc.). I don't mind if we call it `built_table` or `table` though. So if we prefer that, it would be nice if we can do that for the rest of the tests also. Do you have any preference for this one?

Otherwise I would stick with `table `and change the remaining spot where you have it called `built_table`.

Reply:
Oh yes, let's call it  just `table` then and change the other case that doesn't match that. Agree that we should only use `built` if it's really necessary to disambiguate from other things.